takeshape/penny
# Automatically delete PR branches when PR closed

### Screenshots

![Screen_Shot_2022-11-28_at_11_28_37_AM](https://user-images.githubusercontent.com/211047/204368290-e4020913-7c15-427b-9470-9c1c677276db.png)

### Test Plan (Steps to test):

1. I tested this by making a private repo and adding my `cleanup-pr-branch` Github action to it.
1. If you test it this way, note the required secrets in the README and add them to your repo.
1. Commit the action to master
1. Create a TakeShape branch on your test project and a Github branch with the same name
1. Make sure deleting the Github branch kicks off the action and deletes the TS branch
1. Delete the repo when you're done.

### Checklist:
